### DISCUSSION TITLE
Scroll events not working inside drawer

### DISCUSSION THREAD

**Discussion by OlcayGoeren** [saadeghi/daisyui#1610] [Q&A] [answered]
I want to implement a ScrollSpy Component. 
I have attached a simple scroll event to my drawer component, but it's not printing anything to the console.
```
`useEffect(() => {
    const handleScroll = (event: any) => {
      console.log("window.scrollY", window.scrollY);
    };

    window.addEventListener("scroll", handleScroll);

    return () => {
      window.removeEventListener("scroll", handleScroll);
    };
  }, []);`

```
Any ideas why it's not triggered?

I'm using:
 - react 18
 - daisyui ^2.50.0

**Comment by saadeghi** [comment, marked as answer]
`window` is not getting scrolled using daisyUI drawer. Instead `drawer-content` is getting the scroll.  
- So instead of `window.scrollY` use `document.querySelector('.drawer-content').scrollTop `
- And instead of `window.addEventListener` use `document.querySelector('.drawer-content'). addEventListener`  

It should work this way
